src/main/java/com/rabbitmq/client/RpcClient.java

```diff
@@ -389,14 +389,6 @@ public String getRoutingKey() {
         return _continuationMap;
     }
 
-    /**
```

Comment:
This is a breaking change. The last correlation ID must be kept in some way, likely by keeping the `_correlationId` property and update it in the synchronized block.